app/views/fragments/checkout/reviewPanel.scala.html

```diff
@@ -15,7 +15,7 @@
 <div class="review-panel">
 
     <input type="hidden" name="subscriptionDetails"
-           data-amount="@subscription.firstPlan.charges.gbpPrice.amount"
+           data-amount="0"
```

Comment:
@paulbrown1982  gbpprice was used here as well, is it ok to set it to 0 here also?